Formula/webp-pixbuf-loader.rb

```diff
@@ -0,0 +1,69 @@
+class WebpPixbufLoader < Formula
+  desc "WebP Image format GdkPixbuf loader"
+  homepage "https://github.com/aruiz/webp-pixbuf-loader"
+  url "https://github.com/aruiz/webp-pixbuf-loader/archive/0.0.4.tar.gz"
+  sha256 "cd6e4ec44755e8df3e298688c0aeb72b9467bbdd03009989c0d94b219b30fb51"
+  license "LGPL-2.0-or-later"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "gdk-pixbuf"
+  depends_on "webp"
+
+  # Constants for gdk-pixbuf's multiple version numbers.
+  def gdk_so_ver
+    "2.0"
+  end
+
+  def gdk_module_ver
+    "2.10.0"
+  end
+
+  # Subfolder that pixbuf loaders are installed into.
+  def module_subdir
+    "lib/gdk-pixbuf-#{gdk_so_ver}/#{gdk_module_ver}/loaders"
+  end
```

Comment:
I don't think I like the idea of maintaining these constants separately from `gdk-pixbuf`. Can we find a way to determine these dynamically instead?

Reply:
The `2.10.0` version looks like it could be determined from pkg-config, but I don't know of a way to get the 2.0 version.

However, the `gdk-pixbuf` formula maintains the same constants, so would just doing `Formula["gdk-pixbuf"].gdk_so_ver` be acceptable?

Reply:
I'm personally fine with that, but I think there were some objections to similar use of `Formula["gcc"].version_suffix`. Not aware of the reasons though. Maybe @Homebrew/core remembers.

Reply:
I think my objections have generally been around "would changing that in the gdk-pixbuf formula in a way that breaks webp-pixbuf-loader break in CI". If we're confident the answer is "yes": I'm fine with it. If not: I'm not 😁

Reply:
Thanks! I've verified that `brew test webp-pixbuf-loader` does fail if either of these constants are wrong